apps/web/src/components/auth/githubUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { SignUpOriginEnum } from '@novu/shared';
```

Comment:
Filename should be `gitHubUtils`, so I can be coherent with my previous comments. 🙂